openapi/pex-openapi.yaml

```diff
@@ -351,17 +351,21 @@ components:
         format:
           type: string
           description: The Proof or JWT algorithm that the proof is in
+        cryptosuite:
+          type: string
+          description: The data integrity cryptosuite
```

Comment:
Why do we need to add this at all to the descriptor? The descriptor is based on the spec, and only the format specific object should change I think.

I think a format entry of di_vp/di_vc should be enough, then you can further process the vc/vp knowing that you should process the proof object and optionally the cryptosuite

Reply:
Okay, then let's go this way